packages/sit-onyx/src/components/OnyxListbox/OnyxListbox.vue

```diff
@@ -79,8 +145,23 @@ const {
       >
         {{ option.label }}
       </OnyxListboxOption>
+
+      <li v-if="loadingMode === 'button' && !isEmpty" class="onyx-listbox__slot">
```

Comment:
we're already in an v-else block which excludes isEmpty to be truthy, so this is redundant

```suggestion
      <li v-if="loadingMode === 'button'" class="onyx-listbox__slot">
```